SimTKcommon/include/SimTKcommon/internal/Array.h

```diff
@@ -43,6 +43,7 @@
 #include <type_traits>
 #include <initializer_list>
 #include <utility>
+#include <iostream>
```

Comment:
Is this needed?

Reply:
No, sorry, this was just for debugging and I did not remove it.